rllib/env/utils.py

```diff
@@ -47,6 +47,16 @@ def _gym_env_creator(env_context: EnvContext, env_descriptor: str) -> gym.Env:
     # Try creating a gym env. If this fails we can output a
     # decent error message.
     try:
-        return gym.make(env_descriptor, **env_context)
+        # Special case: Atari not supported by gymnasium yet -> Need to use their
```

Comment:
Making sure, this works properly with the new "ALE/"-prefixed Atari identifiers.

Reply:
gotcha.